pkg/scripts/render.go

```diff
@@ -83,6 +83,7 @@ var (
 			{{ template "container-runtime-daemon-config" . }}
 			{{ template "containerd-systemd-setup" . -}}
 			sudo systemctl enable --now docker
+			systemctl status kubelet && sudo systemctl restart kubelet
```

Comment:
Why do we need `systemctl status kubelet` here? To check does the service exists?

Reply:
Yep, at the time if the initial install kubelet service doesn't exist yet.